MAINTAINERS.md

```diff
@@ -77,8 +87,73 @@ Make frequent project releases to the community.
 
 ### Promote Other Maintainers
 
-Assist, add, and remove [MAINTAINERS](MAINTAINERS.md). Exercise good judgement, and propose high quality contributors to become co-maintainers.
+Assist, add, and remove [MAINTAINERS](MAINTAINERS.md). Exercise good judgement, and propose high quality contributors to become co-maintainers. See [Becoming a Maintainer](#becoming-a-maintainer) for more information.
 
 ### Describe the Repo
 
 Make sure the repo has a well-written, accurate, and complete description. See [opensearch-project/.github#38](https://github.com/opensearch-project/.github/issues/38) for some helpful tips to describe your repo.
+
+## Becoming a Maintainer
+
+You can become a maintainer by actively [contributing](CONTRIBUTING.md) to any project, and being nominated by an existing maintainer.
```

Comment:
The criteria to nominate someone can vary greatly between maintainers. How do we make sure that the maintainers are not constantly blocking each other's nominees?

Reply:
I think @nknize mention he'd seen something similar, and that people would basically just re-nominate (there's nothing to stop you from nominating someone again).  I'd be curious to hear other's thoughts on better guardrails.

Reply:
I think it's absolutely OK. There can always be a maintainer that vetoes everything all the time. There can always be someone nominating random people. Any group of people needs to find a way to productively disagree, and YMMV. 

In my experience, those who became maintainers have had an opportunity to earn trust in a way that doesn't prevent progress, so it won't happen often. And we have an escalation path to the admins for resolving this if it happens.

Reply:
I agree with @dblock, the similar consensus rules are used by ASF and OpenJDK for example, and usually the process is driven by good will (whenever there are a -1/no votes, the compelling reasons are expected to be given, I saw that only few times in many years).